### PR TITLE
Feat(welcome modal FWLT-189)

### PR DESCRIPTION
**Pull request by jjBlockchain**

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/components/Flyout/types.ts [hunk 1]
```diff
@@ -1,2 +1,6 @@
 // export type E2EType = 'borrowModal' | 'xyzModal'
-export type E2EType = 'borrowModal' | 'faqModal' | 'whatsNewModal'
+export type E2EType =
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/components/Flyout/types.ts]: Very nice

**Comment by plondon** [comment]
We can't merge this until we're ready to launch simple buy right?

**Comment by schnogz** [comment]
We are merging this into the `feat/simple-buy` branch not `development` for that exact reason

**Comment by plondon** [comment]
Didn't notice that, beautiful

**Pull request merged**